### PR TITLE
Add shared snapshots for grafana

### PR DESCRIPTION
**Pull request by aleksbykov**
Save and store on external service interactive grafana snapshots

### CODE REVIEW

**Comment by bentsi** [comment]
Please resolve conflicts

File: utils/share_snapshot.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+"use strict";
```
Comment (bentsi) [inline utils/share_snapshot.js]: move to `data_dir`

Comment (aleksbykov) [reply utils/share_snapshot.js]: done

**Comment by roydahan** [comment]
Why doesn't it have a backport label?
@bentsi if we want it working for performance we need to backport to branch-perf-v5.

**Comment by bentsi** [comment]
the branch is too old(4months) to backport to it... I think we need to create new perf branch

**Comment by roydahan** [comment]
If we can backport manually that's also fine.
If we create a new branch we need to make sure past results doesn't change.

**Comment by bentsi** [comment]
What do you mean past results doesn't change?

**Comment by roydahan** [comment]
I mean it doesn't affect on anything we run now and isn't comparable to the future runs we will do with newer code.
Low chances but still want to make sure.

**Comment by bentsi** [comment]
the code that we push should be backward compatible... @aleksbykov ?

**Comment by aleksbykov** [comment]
yes, code should be backward compatible.There could be some conflicts with ES structure and fields name, but only fields store links. ( affect on report if we run compare for old test run). But checks are added for correct processing.

**Comment by bentsi** [comment]
@roydahan do you want me to create new perf branch?

**Comment by roydahan** [comment]
ok, please create v6.
please keep v5 but update the labels and the wiki page.

**Comment by bentsi** [comment]
created `branch-perf-v6` from latest master

**Comment by roydahan** [comment]
I updated the perf jobs to use v6, I'll run them this week and we will see if it's good.

**Pull request merged**